features/features.json

```diff
@@ -1013,15 +1013,8 @@
     "type": ["Editor"]
   },
   {
-    "title": "Round Profile Pictures",
-    "description": "All profile pictures on the Scratch website will be rounded.",
-    "credits": ["Scratchfangs", "rgantzos"],
-    "urls": [
-      "https://scratch.mit.edu/users/scratchfangs/",
-      "https://scratch.mit.edu/users/rgantzos/"
-    ],
-    "file": "round-profile-pictures",
-    "type": ["Website", "Theme"]
+    "id": "round-profile-pictures",
+    "version": 2
```

Comment:
This feature was previously bugged and has a few super cool new changes with this update, so let's move this to the top of the list as well.